### ISSUE TITLE
Redundant import in store unit test

### ISSUE THREAD

**Issue by doodlewind** [toeverything/blocksuite#222]
The `pnpm test:unit` command throws the `Yjs was already imported` issue:

```
➜  blocksuite git:(master) pnpm test:unit

> blocksuite@ test:unit /Users/ewind/code/blocksuite
> vitest --run


 RUN  v0.23.4 /Users/ewind/code/blocksuite

stderr | unknown test
Yjs was already imported. This breaks constructor checks and will lead to issues! - https://github.com/yjs/yjs/issues/438

 ✓ packages/store/src/__tests__/jsx.spec.ts (3)
 ✓ packages/store/src/__tests__/space.spec.ts (15)

Test Files  2 passed (2)
     Tests  18 passed (18)
  Start at  08:44:04
  Duration  670ms (transform 321ms, setup 0ms, collect 185ms, tests 29ms)
```

**Comment by himself65** [comment]
I think it already disappeared in the latest branch

**Comment by doodlewind** [comment]
> I think it already disappeared in the latest branch

I still reproduced this using a fresh clone of the latest master, note that here we are running `pnpm test:store` instead of `pnpm test`.

**Comment by doodlewind** [comment]
I think this is fixed by the Yjs update in #508
